clients.go

```diff
@@ -63,6 +64,45 @@ func (u *Unifi) GetClientsDPI(sites []*Site) ([]*DPITable, error) {
 	return data, nil
 }
 
+// GetClientHistory pulls client history data from the controller.
+func (u *Unifi) GetClientHistory(sites []*Site, opts *ClientHistoryOpts) ([]ClientHistory, error) {
+	if opts == nil {
+		opts = NewClientHistoryOpts()
+	}
+	params := url.Values{}
+	params.Add("onlyNonBlocked", fmt.Sprint(opts.OnlyNonBlocked))
+	params.Add("includeUnifiDevices", fmt.Sprint(opts.IncludeUnifiDevices))
+	params.Add("withinHours", fmt.Sprint(opts.WithinHours))
+	paramStr := params.Encode()
```

Comment:
This string is only used in one place; recommend relocating `params.Encode()` to that place.

Reply:
The single place that params.Encode() is used is within a for loop ranging **sites.** 

Would it be beneficial to update this?